cli/bpmetadata/schema/generate.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	_ "embed"
```

Comment:
It doesn't look like we use the embed directive in this file?

Reply:
Removed